### PR TITLE
Minor correction

### PR DESCRIPTION
**Pull request by ppriyank**
`output = tf.matmul(attention_weights, v)  # (..., seq_len_v, depth_v)`

should be 

output = tf.matmul(attention_weights, v)  # (..., seq_len_q, depth_v)

### CODE REVIEW

**Comment by lamberta** [comment]
Thanks.
This diff isn't too good. Can you try saving from Colab?
https://colab.research.google.com/github/ppriyank/docs/blob/pathak/site/en/r2/tutorials/text/transformer.ipynb

**Comment by ppriyank** [comment]
I signed it!

**Comment by ppriyank** [comment]
> Thanks.
> This diff isn't too good. Can you try saving from Co

I should have made all changes in one pull request. Apologies for that. There is another pull request running alongside this, which is taken care of by @yashk2810 I will do another Pull request will all corrections in one go, if I find any else will try there on google collab directly.

**Comment by lamberta** [comment]
Hi @ppriyank, I've confirmed you signed the CLA but it looks like you used a different email address for these commits so the CLA bot is getting confused.
See the GitHub guide [Setting your commit email address in Git](https://help.github.com/en/articles/setting-your-commit-email-address-in-git).
Thanks

**Comment by yashk2810** [comment]
Hey, the change you made in PR (https://github.com/tensorflow/docs/pull/658) somehow got into this PR. Can you revert that change please?

**Comment by ppriyank** [comment]
> Hey, the change you made in PR (#658) somehow got into this PR. Can you revert that change please?

I'm going to do another PR, just a minute, that will have some more corrections and revert the earlier one. Apologies for the mistake

File: site/en/r2/tutorials/text/transformer.ipynb [hunk 1]
```diff
@@ -743,16 +647,18 @@
         "\n",
         "  # add the mask to the scaled tensor.\n",
         "  if mask is not None:\n",
-        "    scaled_attention_logits += (mask * -1e9)\n",
+        "    scaled_attention_logits += tf.log(tf.cast(mask ,  dtype = tf.float32 ))\n",
```
Comment (lamberta) [inline site/en/r2/tutorials/text/transformer.ipynb]: This should not change.

File: site/en/r2/tutorials/text/transformer.ipynb [hunk 2]
```diff
@@ -98,29 +129,29 @@
...
         "colab_type": "code",
-        "id": "V_sMDzDT9x4r"
+        "colab": {}
       },
-      "outputs": [],
       "source": [
         "from __future__ import absolute_import, division, print_function, unicode_literals\n",
         "\n",
-        "!pip install tensorflow-gpu==2.0.0-beta0\n",
+        "!pip3 install tensorflow-gpu==2.0.0-beta0\n",
```
Comment (lamberta) [inline site/en/r2/tutorials/text/transformer.ipynb]: Keep as pip

**Comment by ppriyank** [comment]
I tried to revert all mistakes, can anyone please  have a look
the only  change right now is the **seq_len_v** replaced with **seq_len_q** in the comments

**Pull request merged**